### PR TITLE
Probe model adjustment

### PR DESCRIPTION
**Pull request by Ollieollieolllie**
Fixed gravity vector orientation as well

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/animation/fn_probes.sqf [hunk 1]
```diff
@@ -23,7 +23,7 @@ if !(player == currentPilot _heli) exitwith {};
 private _realRPM = [_heli] call fza_sfmplus_fnc_getRtrRPM;
 private _inducedFlow = [0.0, 0.0, _realRPM * 1.0];
 ([_heli, fza_ah64_sfmplusEnableWind] call fza_sfmplus_fnc_getVelocities) params ["","","","","_velModelSpace"];
-private _gravity = _heli worldtomodelVisual (getpos _heli vectoradd [0,0,-9.8]);
+private _gravity = _heli worldtomodelVisual (getpos _heli vectoradd [0,0,9.8]);
```
Comment (BradMick) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/animation/fn_probes.sqf:26]: Just make this a 1 value, you'll also want to do a sin and cos function using the pitch/bank of the helicopter to influence it as well.

**Comment by mattysmith22** [comment]
I'll leave this with Brad on the physics modelling, if you two are happy, I'm happy

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/animation/fn_probes.sqf (hunk 1 above)
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/animation/fn_probes.sqf]: iv changed the value to 1, sin & cos are not necessary, I get the gravity vector by translating world to model space

**Review by BradMick — COMMENTED** [review]
So, this works for now. It should be able to be a bit more dynamic once proper induced flow velocity is added. I'm not 100% that this should be in the SFM+/HeliSim bit of the code though. HeliSim doesn't handle animations of any kind.

**Comment by Ollieollieolllie** [comment]
i wasn't to sure where to stick this bit of code, il get matty's opinion

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/animation/fn_probes.sqf (hunk 1 above)
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/animation/fn_probes.sqf]: Updated

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/animation/fn_probes.sqf [hunk 2]
```diff
@@ -23,7 +23,7 @@ if !(player == currentPilot _heli) exitwith {};
 private _realRPM = [_heli] call fza_sfmplus_fnc_getRtrRPM;
 private _inducedFlow = [0.0, 0.0, _realRPM * 1.0];
 ([_heli, fza_ah64_sfmplusEnableWind] call fza_sfmplus_fnc_getVelocities) params ["","","","","_velModelSpace"];
-private _gravity = _heli worldtomodelVisual (getpos _heli vectoradd [0,0,-9.8]);
+private _gravity = _heli worldtomodelVisual (getpos _heli vectoradd [0,0,1]);
```
[the remark this answers is not in the document]
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/animation/fn_probes.sqf]: Done

**Review by mattysmith22 — APPROVED** [review]

**Pull request merged**